src/app/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import Providers from './Providers';
+import './globals.css';
+
+export const metadata = {
+  title: 'MUWITH',
+  description: 'Now, play music that fits your mood and mood anywhere.',
+};
+
+export default function RootLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
```

Comment:
```suggestion
}: React.PropsWithChildren) {
```
여기도 `React.PropsWithChildren`을 사용해서 더 깔끔하게 정리하는것이 좋을것같습니다.

똑같은 이유로, `src\app\(auth)\layout.tsx`와 `src\app\(main)\layout.tsx` 부분도 수정이 이루어지면 좋지 않을까 싶어요.